lib/osf-components/addon/components/subscriptions/list-row/styles.scss

```diff
@@ -0,0 +1,22 @@
+.SubscriptionRow {
+    display: flex;
+    justify-content: space-between;
+    width: 75%;
+    height: 45px;
+    border-top: 1px solid $color-bg-gray-dark;
```

Comment:
could add `align-items: center` here and remove `align-self` from the children (. RowEventNameSection and . RowPowerSelectSection)